src/Domain/Bootstrap.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Contains the Bootstrap class
+ *
+ * @package WooCommerce/Blocks
+ */
+
+namespace Automattic\WooCommerce\Blocks\Domain;
+
+use Automattic\WooCommerce\Blocks\Assets as OldAssets;
```

Comment:
Ya I already have an issue for tracking the removal: https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/973

Since I plan to do that fairly soon after this pull gets merged I'm not sure we need to do the name change right now (Ideally I'd like to get the refactor complete before next plugin release).

Regarding using `@todo`.  It appears our precommit hook flags those and prevents commits.  Is that something intended, or should we do a pull to tweak that?

Reply:
cc @Aljullu 

I think we should allow it. I got moaned at for using `TODO:` syntax before because of that rule :p

Reply:
> I think we should allow it. I got moaned at for using TODO: syntax before because of that rule :p

It sounds good to me removing that rule. It looks like neither WooCommerce nor WooCommerce Admin have it (they have plenty of `@todo` comments around the code), so I guess we should copy them.

Anyway, I still think if we can handle to-do tasks with GitHub issues, it's better than in-line comments.

Reply:
See https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/984